com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -2195,13 +2195,25 @@ public IResource getGeneratedResource(long hash, String suffix) {
     }
 
     public IResource createGeneratedResource(long hash, String suffix) {
+        return createGeneratedResource(null, hash, suffix);
+    }
+
+    public IResource createGeneratedResource(String prefix, long hash, String suffix) {
```

Comment:
An ability to specify `prefix`, it's up to developer to make sure it's unchanged between generations.